floodviz/map_utils.py

```diff
@@ -2,15 +2,33 @@
 
 def site_dict(site_list, url_prefix):
     """Puts site data into a dictionary
-
     Args:
         site_list: A list of site ids to be queried on NWIS
-        url_prefix: The beginning of the NWIS url
-
+        url_prefix: A string containing the beginning of the NWIS site url
     Returns:
-        A dict containing various site information in a usable format.
+        An array of dicts containing various site information in a usable format.
     """
 
+    if not site_list:
```

Comment:
So your function comments say that it returns an array of dictionaries. If an empty list of sites is passed in, then return an empty array. I don't think this is actually an error condition. You could print out an info statement but it's not really an error.